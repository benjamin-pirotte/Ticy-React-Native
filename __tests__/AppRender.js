import ReactNative from 'react-native';
import React from 'react';
import AppIos from '../build/index.ios.js';
import AppAndroid from '../build/index.android.js';
import renderer from 'react-test-renderer';

it('Ios renders correctly', () => {
  const tree = renderer.create(
    <AppIos />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('Android renders correctly', () => {
  const tree = renderer.create(
    <AppAndroid />
  ).toJSON(); 
  expect(tree).toMatchSnapshot();
});
