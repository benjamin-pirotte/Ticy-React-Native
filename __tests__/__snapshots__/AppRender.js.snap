exports[`test Android renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }>
  <View
    style={
      Object {
        "flex": 1,
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      Loading
    </Text>
    <View />
  </View>
</View>
`;

exports[`test Ios renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }>
  <View
    style={
      Object {
        "flex": 1,
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      Loading
    </Text>
    <View />
  </View>
</View>
`;
